Kbs.Wpf/User/Update/AccountView.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<Page x:Class="Kbs.Wpf.User.Update.AccountView"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:Kbs.Wpf.User.Update"
+      mc:Ignorable="d" 
+      Background="White"
+      d:DesignHeight="450" d:DesignWidth="800"
+      Title="AccountView">
+    <Page.DataContext>
+        <local:AccountViewModel />
+    </Page.DataContext>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <!-- Vaste positie voor de linkerkant -->
+            <ColumnDefinition Width="*" />
+            <!-- Flexibele ruimte voor groei -->
+        </Grid.ColumnDefinitions>
+        <Label Content="Instellingen" HorizontalAlignment="Center" Margin="0,50,0,0" VerticalAlignment="Top" FontWeight="Bold" FontSize="22"/>
+
+        <Label Content="Email:" HorizontalAlignment="Center" Margin="-250,96,0,0" VerticalAlignment="Top" FontWeight="Bold"/>
+        <TextBox Name="EmailInput" HorizontalAlignment="Left" Margin="108,122,0,0" VerticalAlignment="Top" Width="180" Text="{Binding InputEmail}" />
```

Comment:
`Name="EmailInput"` moet weg en vervangen worden met de InputEmail binding. Je hebt de instantie van de label niet in je page-klasse nodig, alleen de waarde ervan. Geldt ook voor andere plekken waar `Name="..."` staat